.github/ISSUE_TEMPLATE.md

```diff
@@ -1,37 +1,27 @@
 <!--
-Questions:
-  https://groups.google.com/forum/#!forum/loopbackjs
-  https://gitter.im/strongloop/loopback
-Immediate support:
-  https://strongloop.com/api-connect-faqs/
-  https://strongloop.com/node-js/subscription-plans/
+HELP US HELP YOU BY PROVIDING AS MUCH OF THE FOLLOWING INFORMATION AS POSSIBLE AND
+AVOID DUPLICATES BY SEARCHING BEFORE CREATING A NEW ISSUE.
 -->
 
-# Description/Steps to reproduce
+## I'm submitting a ...
+<!-- Check one of the following options with a "x" -->
+<pre><code>
+[ ] Regression (a behavior that used to work and stopped working in a new release)
+[ ] Bug report
+[ ] Feature request <!-- Include a problem statement, use case scenario, example, code samples, etc. -->
+[ ] Documentation issue or request
+[ ] Support / Question
+[ ] Feedback / Comment / Thanks / Other
+</code></pre>
```

Comment:
For posterity, I am cross posting from https://github.com/strongloop/loopback-next/issues/979#issuecomment-368810473:

> Please don't use checklists as a replacement for radio buttons. GitHub shows completed + total number of items from checklists in GitHub issue list (see [list of pull requests](https://github.com/strongloop/loopback-next/pulls) to see yourself). We don't want every user-reported issue to say "1 of 3" in the listing.
>
> Regarding issue type in particular, we used to have a checklist like that in the past and it did not work well. In my experience, users often don't know what is the type of their report. Things that look like a bug to them may be considered as a missing feature by us. Discussions that start like a question can often evolve into a bug report or a feature request.

IIUC, wrapping the check boxes in `<pre><code`> will prevent GitHub from treating this content as a checklist. Which is good!

I don't like the rendered output of this formatting, but I can live with it until we find a better solution.

<img width="924" alt="screen shot 2018-03-01 at 10 12 24" src="https://user-images.githubusercontent.com/1140553/36836241-212e1772-1d39-11e8-9c73-ff8049e35c75.png">

Reply:
The idea to use `<pre><code>` came from Angular's Issue template. https://github.com/angular/angular/blob/master/.github/ISSUE_TEMPLATE.md

I saw that comment and agree that we should triage the issue and label it accordingly but I think it may be helpful to know what a user thinks it might be. 

That said ... This is something I'd be ok to remove as well if that's what makes sense. A pro of that would be a simpler template. Thoughts @strongloop/sq-lb-apex?